EnglishFilter/sections/adservers.txt

```diff
@@ -5,6 +5,7 @@
 !
 !! https://a.bestcontentitem.top/warp/*?r=
 !
+||handtub.com^
```

Comment:
Missed `$third-party`.

Reply:
No referer. Please check my commits name

Reply:
We are adding this to avoid broken sites in the future. Also this request has a referer,

Reply:
Ah yes. Just now it wasn't there. I am not sure.